### PR TITLE
Secret Assignment

### PR DESCRIPTION
**Pull request by HeroicHorizon**

### CODE REVIEW

File: secret/cipher/cipher.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	}
+	return fmt.Sprintf("%x", ciphertext), err
+}
+
+// Decrypt it decrypt's the key value
+func Decrypt(key, cipherHex string) (string, error) {
+	var ciphertext []byte
+	block, err := newCipher(key)
+	if err != nil {
+		fmt.Println(err)
```
Comment (MohitArora1) [inline secret/cipher/cipher.go]: we can return from here, when ever you got error return immediately.

Comment (MohitArora1) [reply secret/cipher/cipher.go]: and you can follow this all rest of the places

**Review by MohitArora1 — CHANGES_REQUESTED** [review]

File: secret/cipher/cipher.go (hunk 1 above)
Comment (HeroicHorizon) [reply secret/cipher/cipher.go]: Addressed the comments

**Review by MohitArora1 — APPROVED** [review]
LGTM

**Pull request merged**